frontend/dockerfile/parser/parser.go

```diff
@@ -341,7 +352,11 @@ func Parse(rwc io.Reader) (*Result, error) {
 		}
 
 		if hasEmptyContinuationLine {
-			warnings = append(warnings, "[WARNING]: Empty continuation line found in:\n    "+line)
+			warnings = append(warnings, Warning{
+				Short:    "Empty continuation line found in: " + line,
+				Detail:   "Empty continuation lines will become errors in a future release. https://github.com/moby/moby/pull/33719",
```

Comment:
@thaJeztah Is this actually marked as deprecated somewhere. Can't find it.

Reply:
Hmm.. good one; we should at least have it documented in https://docs.docker.com/engine/deprecated/ (looks like we don't), with more discussion in https://github.com/moby/moby/issues/29005.

I _think_ we decided to not remove it (at least for the time being), but to make the warning a bit more scary. Will have to dig those discussions though.